From 9d980c6da6d37244de2ef3ccdf7bcea0db5a2070 Mon Sep 17 00:00:00 2001
From: Troy Harvey <harveydevel@gmail.com>
Date: Thu, 5 Jun 2025 10:49:27 +1000
Subject: [PATCH] Switch from readfp to read_file

"configparser.ConfigParser no longer has a readfp method. Use read_file() instead."

https://docs.python.org/3/whatsnew/3.12.html

Signed-off-by: Troy Harvey <harveydevel@gmail.com>
---
 ypkg2/main.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ypkg2/main.py b/ypkg2/main.py
index 0d31606..2a8d825 100644
--- a/ypkg2/main.py
+++ b/ypkg2/main.py
@@ -210,7 +210,7 @@ def build_package(filename, outputDir, buildDir=None):
             continue
         try:
             c = configparser.ConfigParser()
-            c.readfp(open(fpath))
+            c.read_file(open(fpath))
             pname = c.get("Packager", "Name")
             pemail = c.get("Packager", "Email")
 
